unittests/API/APITest.cpp

```diff
@@ -231,7 +231,7 @@ TEST_F(HermesRuntimeTest, PreparedJavaScriptInvalidSourceThrows) {
   bool caught = false;
   try {
     rt->prepareJavaScript(std::make_unique<StringBuffer>(badSource), "");
-  } catch (const facebook::jsi::JSIException &err) {
+  } catch (const facebook::jsi::JSIException & /*err*/) {
```

Comment:
Could these use EXPECT_THROW instead? It should simplify them and avoid issues like the one you ran into.

Reply:
Let me try it.

Reply:
I have updated the changed unit tests to use `EXPECT_THROW` instead.